pkgs/servers/sickbeard/patches/override-python-version-check.patch

```diff
@@ -0,0 +1,28 @@
+From afc0e88bd31d83795325792944d1806a68f813b4 Mon Sep 17 00:00:00 2001
+From: rembo10 <rembo10@users.noreply.github.com>
+Date: Tue, 23 Aug 2022 12:43:39 +0530
+Subject: [PATCH] Allow running on unsupported Python versions
+
+---
+ sickgear.py | 5 +----
+ 1 file changed, 1 insertion(+), 4 deletions(-)
+
+diff --git a/sickgear.py b/sickgear.py
+index e90e0f45..1972323e 100755
+--- a/sickgear.py
++++ b/sickgear.py
+@@ -43,10 +43,7 @@ versions = [((2, 7, 9), (2, 7, 18)), ((3, 7, 1), (3, 8, 13)),
+             ((3, 9, 0), (3, 9, 2)), ((3, 9, 4), (3, 9, 13)),
+             ((3, 10, 0), (3, 10, 5))]  # inclusive version ranges
+ if not any(list(map(lambda v: v[0] <= sys.version_info[:3] <= v[1], versions))) and not int(os.environ.get('PYT', 0)):
+-    print('Python %s.%s.%s detected.' % sys.version_info[:3])
+-    print('Sorry, SickGear requires a Python version %s' % ', '.join(map(
+-        lambda r: '%s - %s' % tuple(map(lambda v: str(v).replace(',', '.')[1:-1], r)), versions)))
+-    sys.exit(1)
++    print('Unsupported Python version: %s.%s.%s' % sys.version_info[:3])
```

Comment:
We might as well drop this entirely since it will probably be not looked at in the log.

Reply:
Yeah I think that's right. Should I just remove the sys.exit(1) call, or the whole `if` block? I think it makes sense to keep in the versions array, because it might create merge conflicts down the road....

Reply:
I've pushed the update if you want to check